jobscheduler/scheduler-component-shared/src/java/org/sakaiproject/component/app/scheduler/SchedulerManagerImpl.java

```diff
@@ -745,4 +735,39 @@ public boolean isStartScheduler() {
    public void setStartScheduler(boolean startScheduler) {
        this.startScheduler = startScheduler;
    }
+
+    @Override
+    public void start() {
+        if (isStartScheduler()) {
```

Comment:
Probably should make a call to super() as your overriding the application context start()